.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,35 @@
+name: Bug Report 🐛
+description: File a bug report
+title: '[Bug Report]: '
+labels: ['Bug Report']
```

```suggestion
labels: ['bug']
```
or do we want a new label?